test/specs/login.page.test.js

```diff
@@ -0,0 +1,73 @@
+const { expect } = require("@wdio/globals");
+const LoginPage = require("../pageobjects/login.page");
+const HomePage = require("../pageobjects/home.page");
+const testData = require("../data/testData");
+
+describe("Login Page Tests", () => {
+  beforeAll(async () => {
+    await LoginPage.open();
+  });
+
+  it("should show error messages for email and password fields when both are empty", async () => {
+    await LoginPage.login("", "");
+
+    expect(await LoginPage.isEmailErrorDisplayed()).toBe(true);
+    expect(await LoginPage.isPasswordErrorDisplayed()).toBe(true);
```

Comment:
For all methods related to login functionality, you can create a Utility folder and place all related checks and methods that pertain to login there. For example, you could have a loginUser method that fills in all login data and checks that the login was successful. If we have a negative test case for this, specifically, you can move those checks into a separate method within that utility class and simply call that method here. Additionally, I would separate out these negative login tests into a dedicated JavaScript file and name it negative-login-cases.js.